tests/drivers/i2c/i2c_slave_api/src/eeprom_slave.c

```diff
@@ -11,11 +11,11 @@
 #include <errno.h>
```

Comment:
I'm not very good at reviewing on github yet. But I would expect to see this file added to the CMakeLists.txt file?

Reply:
Most of the Zephyr samples and tests (including this one) have this in `CMakeLists.txt`:
```
FILE(GLOB app_sources src/*.c)
```
so there's no need to list individual sources.

Reply:
OK ta